core/2d/CCParticleSystem.cpp

```diff
@@ -218,14 +242,36 @@ ParticleSystem::ParticleSystem()
     , _startSpinVar(0)
     , _endSpin(0)
     , _endSpinVar(0)
+    , _spawnAngle(0)
+    , _spawnAngleVar(0)
+    , _isHsv(false)
```

Comment:
Maybe this variable can be remove, use `_isSVAllocated` instead

Reply:
True! I didn't think about it.